src/assets/sass/protocol/components/_button.scss

```diff
@@ -41,53 +106,75 @@ a.mzp-c-button {
     }
 }
 
-.mzp-c-button.mzp-t-small {
-    padding: ($spacing-sm - 2px) ($spacing-md - 2px);
+// Sizes
+
+.mzp-c-button.mzp-t-sm {
+    @include font-size(12px);
+    border-radius: $border-radius-xs;
+    padding: 1px (16px - 2px);
 }
 
-.mzp-c-button.mzp-t-dark,
-a.mzp-c-button.mzp-t-dark {
-    background-color: $color-white;
-    border-color: $color-white;
-    color: $color-black;
+.mzp-c-button.mzp-t-small, // backwards compatability
+.mzp-c-button.mzp-t-md {
+    @include font-size(14px);
+    padding: 3.5px (16px - 2px);
```

Comment:
These seem like strange numbers. Can this just be `padding: $spacing-xs $spacing-md`?

Reply:
The specs I have are for a specific button height. Which means that I can't jus- oh wait actually maybe I can. BRB.

Reply:
Okay, I switched to `line-height` to define the height of these and I think that works better.

Reply:
The greater line-height looks wonky if the button label wraps to two lines, which we can't guarantee will never happen. I don't think we need to exactly match the height in the spec to the exact pixel if we're at least close to it with whole numbers.

Reply:
I think the greater line height looks fine.

Reply:
Medium button with the current `line-height: remify(32px - 4px);` (1.75rem) and `padding: 0 (24px - 2px);`:
![image](https://user-images.githubusercontent.com/205591/79493439-57bdb200-7fd6-11ea-8d31-ced0a7347335.png)

With standard `line-height: 1.5` and `padding: $spacing-xs $spacing-lg;`
![image](https://user-images.githubusercontent.com/205591/79493692-b2efa480-7fd6-11ea-86dc-d5cd570dd919.png)

Maybe the buttons wrapping is an edge case. But it also seems like one we can solve for pretty easily and still avoid magic numbers in the CSS. If there's a good reason the medium button has to be *exactly* 32px tall and 33px is unacceptable then I suppose magic numbers are the only way. Other sizes might still need some padding math to compensate for the border width but if we're within a few pixels of the intended size I don't think anyone will complain.

Reply:
@craigcook @stephaniehobson 

> If there's a good reason the medium button has to be exactly 32px tall and 33px is unacceptable then I suppose magic numbers are the only way.

We were using the pixel specific heights just so we can have something consistent to use amongst the designers and our files. No need to try to hack it or use magic numbers; if `33px` or something close, by all means use it. Hope that helps. :)